packages/card/index.tsx

```diff
@@ -33,6 +37,14 @@ function Card(props: CardProps) {
           <Content>
             {meta && meta.length > 0 ? <Meta items={meta} /> : null}
             <Heading>{heading}</Heading>
+            {productBadges &&
+            productBadges?.badges &&
+            productBadges?.badges?.length > 0 ? (
+              <ProductBadges
+                badges={productBadges.badges}
+                appearance={appearance}
+              />
+            ) : null}
```

Comment:
Looks like badges should display below the description.

<img width="614" alt="CleanShot 2022-10-05 at 17 01 24@2x" src="https://user-images.githubusercontent.com/825855/194162843-3670a2c5-54b2-4305-9026-6df463e581e6.png">

Reply:
Oops - you're right. [Moved](https://github.com/hashicorp/react-components/pull/748/commits/fbb86263b5577426c921f2bc54d978c61a3afd48)!